library/globnames.ml

```diff
@@ -31,6 +31,8 @@ let destConstRef = function ConstRef ind -> ind | _ -> failwith "destConstRef"
 let destIndRef = function IndRef ind -> ind | _ -> failwith "destIndRef"
 let destConstructRef = function ConstructRef ind -> ind | _ -> failwith "destConstructRef"
 
+let subst_inductive = subst_ind
```

Comment:
Why add this?

Reply:
I need it here https://github.com/coq/coq/pull/13958/files#diff-c88255e2c16da26e5792e8aaf5867766695be79e640fa91dff9cf08499a8c919R77
and also makes sense, since the API provides subst for the other globref cases

Reply:
(you need to unfold the diff on structures.ml otherwise the pointer does not work well

Reply:
But subst_ind already exists, let's not add random aliases. If you want consistency you should move the other globref cases instead.

Reply:
Just tell me where and I'll do it

Reply:
where subst_ind is defined

Reply:
(globnames did not seem too bad, I thought subs_ind has to be declared elsewhere for another reason)

Reply:
done